lessons/Algebra2/exploring-linear-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,265 @@
+Strength indicates how closely the two variables are correlated.
+
+How well does knowing the x-value allow us to predict what the y-value will be?
+
+@right{@image{images/A.gif, 200}}**A relationship is strong if knowing the x-value of a data point gives us a very good idea of what its y-value will be** (knowing a student's age gives us a very good idea of what grade they're in). A strong linear relationship means that the points in the scatter plot are all clustered _tightly_ around an invisible line.
+
+@clear
+
+@right{@image{images/1a.gif, 200}}**A relationship is weak if x tells us little about y** (a student's age doesn't tell us much about their number of siblings). A weak linear relationship means that the cloud of points is scattered very _loosely_ around the line.
+
+@clear
+
+
+=== Investigate
+
+Now that you've dug into the role the form, direction and strength play in quantifying a correlation, it's time to put those concepts to work!
+
+[.lesson-instruction]
+In pairs or small groups, complete @printable-exercise{pages/identifying-form-matching.adoc}.
+
+Review student answers, and have students _explain their thinking_ for this activity. For students who are struggling, hearing what their peers are looking for is especially helpful at this stage.
+
+[.lesson-instruction]
+In pairs or small groups, complete @printable-exercise{pages/identifying-form.adoc}
+
+Review student answers. Some of the answers are not so clear-cut, and students may disagree about what constitutes a "strong" vs. "weak" correlation. We've tried to choose scatter plots that clearly fall into one category or the other, but without diving into the algorithm for linear regression students may find this exercise somewhat subjective.
+
+[.lesson-instruction]
+Return to @printable-exercise{exploring-states.adoc}, and complete the rest of the page.
+
+
+=== Common Misconceptions
+- Students often conflate strength and direction, thinking that a strong correlation _must_ be positive and a weak one _must_ be negative.
+- Students may also falsely believe that there is ALWAYS a correlation between any two variables in their dataset.
+- Students often believe that strength and sample size are interchangeable, leading to mistaken assumptions like "any correlation found in a million data points _must_ be strong!"
+
+
+=== Synthesize
+
+- What relationships did you find in the states dataset?
+- What was their form, direction, and strength?
+- Were any of these relationships a surprise? Why or why not?
+
+
+== Fitting Linear Models @duration{25 minutes}
+
+=== Overview
+
+The line of best fit is framed as a _predictor function_, which attempts to predict where a new point would fall on the plane based on the relationship in the data. Students define their predictors (linear functions) to find the line of best fit, using @vocab{R-squared} to determine fitness and making predictions with the result. They explore the impact that slope and and y-intercept have on fitness.
+
```

Comment:
I doubt that you meant to write "fitness" here.

Reply:
Fixed (the proper term is "goodness of fit", but _man_ does that seem more cumbersome)